### PR TITLE
Ensure rest/grpc services are running

### PR DESCRIPTION
**Pull request by lampajr**
Changes
- [x] Ensure rest/grpc services are started up before completing the `setup-scripts` phase
- [x] Set full mongo image
- [x] Capture logs from datasources

### CODE REVIEW

File: superheroes.yaml [hunk 1]
```diff
@@ -495,17 +580,20 @@ states:
 
   # Fights
   FIGHTS_ENABLED: true
-  FIGHTS_DB_IMAGE: "mongo:7.0"
+  FIGHTS_DB_IMAGE: "docker.io/library/mongo:7.0"
```
Comment (johnaohara) [inline superheroes.yaml]: Is this image anywhere else? we will be throttled from docker.io, maybe we should publish to quay.io

**Review by johnaohara — CHANGES_REQUESTED** [review]
Looks good, apart from the mongodb image on docker.io : 

`FIGHTS_DB_IMAGE: "docker.io/library/mongo:7.0"`

File: superheroes.yaml (hunk 1 above)
Comment (lampajr) [reply superheroes.yaml]: I could not find the corresponding image on quay.io 🫤 

> we will be throttled from docker.io

true, as long as we are not logged in with a docker account

We might have the same problem for `jaeger`, `opentelemtry` and `postgres`

Comment (johnaohara) [reply superheroes.yaml]: I think we can push images to quay.io for our testing

Comment (lampajr) [reply superheroes.yaml]: As discussed, I updated this PR to use the google mirror `mirror.gcr.io` as default registries for some of the images, like `postgres`, `mariadb` and etc.

Comment (johnaohara) [reply superheroes.yaml]: looks good ty

**Review by johnaohara — APPROVED** [review]
LGTM

**Pull request merged**